### ISSUE TITLE
[Sticky] 🎉🎉Who uses Kitten Tricks / feedback? 🎉🎉

### ISSUE THREAD

**Issue by artyorsh** [akveo/kittenTricks#243]
Hi All! 

We are really keen to collect your feedback, and to start with, we'd like to know who is using Kitten Tricks / UI Kitten. We would really appreciate if you:

- Share your company name
- Share your logo (in which case it might be used in the readme / frontpage etc)
- What kind of project? How large? (code base size or team size)
- If your project is open source or public, please share a link!
- What do you like most about Kitten Tricks, what could be done better?

Thanks to MobX for Questionnaire idea: mobxjs/mobx#681

UI Kitten discussion https://github.com/akveo/react-native-ui-kitten/issues/657

**Comment by kockok** [comment]
Will this demo app be updated to v4.4.0 and be written in functional components with react hooks?

**Comment by artyorsh** [comment]
@kockok yes, it's [in progress](https://github.com/akveo/kittenTricks/pull/254)

**Comment by sudomann** [comment]
I based my project off Kitten Tricks, and have gotten so comfortable with class based components. It was my very first React project so I might be lacking understanding of something; I'm curious why did you guys suddenly choose to switch to functional components and hooks?

**Comment by artyorsh** [comment]
@sudomann thanks for the interest. We try keeping this application trending and up-to-date. Functional components and hooks in today's state of React and React Native is something you should not avoid.

**Comment by sudomann** [comment]
Current technique for changing app theme seems to unmount the entire app?

Building with expo (like kitten tricks), expo simply restarts the app.

But without expo the app just disappears and leaves you on the phone's home screen.

**Comment by sudomann** [comment]
Actually, I've found that it's not caused by something new from the recent refactor of the app. I discovered the issue above after I switched system display theme to see the app switch theme but instead it suddenly disappeared.


Turns out this issue already existed with the previous Kitten Tricks release, I just never noticed it. Tested with a new simple app using ui-kitten and saw the same issue. It seems to affect Android only.

I suspect it's similar to what causes this as well:
https://github.com/akveo/react-native-ui-kitten/issues/593

**Comment by artyorsh** [comment]
@sudomann am I right that you're trying to achieve changing theme behavior using system theme?
If so, I can prove it works properly in Kitten Tricks - [see demo](https://i.imgur.com/UfEJv3V.gif)
[You might be missing `userInterfaceStyle` property in your app.json](https://docs.expo.io/versions/latest/sdk/appearance/#configuration)

**Comment by artyorsh** [comment]
@sudomann I've got this fixed in #261 (your main component should also include `AppearanceProvider` if you're using `react-native-appearance`). Also, let's keep this issue clean and used for feedbacks. I'm going to delete this conversation soon.

Thanks for reporting!

**Comment by chumanhquang** [comment]
@artyorsh,  Will kittenTricks have more screens or a pro version in the future?

**Comment by artyorsh** [comment]
@chumanhquang currently we're thinking about this and it's more likely that yes - it will. Thanks for the interest

**Comment by chumanhquang** [comment]
@artyorsh Thank you, I hope it will soon launch!

**Comment by kuasha420** [comment]
This wonderful project is limited by expo. too many issues after ejecting from expo

**Comment by artyorsh** [comment]
@kuasha420 because you don't need to do this as it is already ejected and [can be runnable in both react-native and expo environments](https://github.com/akveo/kittenTricks/blob/master/DEV_DOCS.md#run).

**Comment by kuasha420** [comment]
OH SHIT THEN DID THAT HAPPEN!! I retract my statement. As I said, It's an wonderful and very modern UI Kit for React Native. I thank you guys wholeheartedly for making it available.

**Comment by mehmetor** [comment]
Hello,

Are you considering improving for @ui-kitten/components v5.0.0 compatibility?

**Comment by heberuriegas** [comment]
@artyorsh What is the best way to start the project and not have the other files interfering with yours? I would like to avoid deleting them 1 by 1. Probably delete a folder instead?
